### ISSUE TITLE
Parser error in Synchronous

### ISSUE THREAD

**Issue by modelica-trac-importer** [tbeu/Modelica_Synchronous#2]
**Reported by sjoelund&#46;se on 15 Apr 2013 11:30 UTC**
Parser error in Synchronous (I would have commited it myself, but I do not have permission):
```diff
Index: Modelica_Synchronous/Examples.mo
===================================================================
--- Modelica_Synchronous/Examples.mo	(revision 6249)
+++ Modelica_Synchronous/Examples.mo	(working copy)
@@ -1586,7 +1586,7 @@
 system within Modelica and a Modelica simulation environment.
 This approach is shown at hand of a simple example using
 a nonlinear plant model of a
-mixing unit (F�llinger O. (1998): Nichtlineare Regelungen I,
+mixing unit (F&ouml;llinger O. (1998): Nichtlineare Regelungen I,
 Oldenbourg Verlag, 8. Auflage, page 279) and utilizing this plant
 model as nonlinear feed-forward controller according to (Looye et.al. 2005):
 </p>
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1076

**Comment by modelica-trac-importer** [comment]
**Comment by thiele on 15 Apr 2013 11:58 UTC**
Fixed in 922d16079706530c566e295b59f1f7e79ae3f71a

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 15 Apr 2013 12:22 UTC**
